app/femr/data/models/mysql/Patient.java

```diff
@@ -164,4 +167,19 @@ public void setIsDeleted(DateTime isDeleted) {
         this.isDeleted = isDeleted;
     }
 
+    @Override
+    public int getDeletedByUserId() {
```

Comment:
This should be Integer, not int to match the type on line 57. If this tries to return null an error will occur because int can't be null (but deleteByUserId can)